fedora/fas.py

```diff
@@ -178,11 +178,11 @@ async def user_hello(self, evt: MessageEvent, username: str | None) -> None:
         """
         await self._user_hello(evt, username)
 
-    @user.subcommand(name="info", help="Return brief information about a Fedora user.")
+    @user.subcommand(name="info", help="Return detailed information about a Fedora user, including username, human name, pronouns, creation date, timezone, locale, and GPG key IDs.")
```

Comment:
I'm not sure if this repository enforces a line limit, but even if it doesn't it would be nice to break this up into multiple lines. You can split up text into multiple source code lines without inserting newlines by doing something like
```
@user.subcommand(
    name="info",
    help=(
        "Return detailed information about a Fedora user, including username,"
        "  human name, pronouns, creation date, timezone, locale, and GPG key IDs."
    ),
)
```

Reply:
Oh? I thought didn't think too much too it. Thank you so much for reviewing, I will make the corrections now.

Reply:
@jeremycline I just made the corrections, thank you again for taking your time to look at my PRs. I am grateful.